mesa/space.py

```diff
@@ -391,7 +391,6 @@ def iter_cell_list_contents(
         # iter_cell_list_contents returns only non-empty contents.
         return (self.grid[x][y] for x, y in cell_list if self.grid[x][y])
 
-    @accept_tuple_argument
```

Comment:
I can merge without this line change. I think we can discuss this later.

Reply:
not a problem

Reply:
reverted